Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -439,16 +439,23 @@ std::pair<DateAndTime, DateAndTime>
 firstLastPulseTimes(::NeXus::File &file, Kernel::Logger &logger) {
   file.openData("event_time_zero");
 
-  if (!file.hasAttr("offset"))
-    throw std::runtime_error("No ISO8601 offset attribute provided");
-
   const auto heldTimeZeroType = file.getInfo().type;
   // Nexus only requires event_time_zero to be a NXNumber, we support two
   // possibilities for held type
 
   std::string isooffset; // ISO8601 offset
-  file.getAttr("offset", isooffset);
-  DateAndTime offset(isooffset);
+  DateAndTime offset;
+  // According to the Nexus standard, if the offset is not present, it implies
+  // the offset is and absolute timestamp, which is relative to the start of
+  // Unix epoch (https://manual.nexusformat.org/classes/base_classes/NXlog.html)
+  if (!file.hasAttr("offset")) {
```

Comment:
Offset for `NxLog`? Do you mean `scaling_factor`?

Reply:
Actually i think i slightly misunderstood what Matt said. He said that when you are dealing with `NXlog`, if there is no reference provided, you assume it's Unix epoch. And we should basically suggest the standard should do the same for `NXevent_data`.
He is saying we should also submit a PR to the Nexus documentation pages.

Reply:
The "NXlog" "time" dataset has an attribute called "start", which has the same function as "offset" for "event_time_zero". The "NXlog" documentation specifically says that if "start" is missing then times are relative to the unix epoch. I believe the same should be true for "offset" in "NXevent_data", and will create a PR to update the NeXus documentation.

Reply:
The code change here LGTM.